preprocess.py

```diff
@@ -113,14 +109,91 @@ def preprocess(args):
         )
     elif args.dataset == "shakespeare":
         device, kwargs = use_device(args)
-        train_dataset = ShakeSpeare(train=True)
+        trainset = ShakeSpeare(train=True)
```

Comment:
LOL, this is [trainset](https://www.google.com/search?sca_esv=14770d8a9caa6d0f&sxsrf=ACQVn08EXj0lLldLlCOCTT2lYM5Ho8Ifjw:1714714960977&q=trainset&uds=AMwkrPs4weTB3nAhxTiOjI0cLaoUD0SWCQBPIAWRLyCpbGKOyTmoaGdJ9DZkPDWjuX5xhbqWsfO062xbows9npjqol77oQisXcKfxIwydZ-agsuJcGIZpXNvLifTxBew2dPseQ4tvRCYhIzDhi6H3L-m19QEhYG2_QITBquv3tQ21NbUAExwyWANWZXJd2uDAPgbi3SvMO3cOCjhY1v8gqMInVz3MlyyA-bI2_5z-Uo_X0B-en2ErKAS22XiTq4_5vh5N23WsOUMIzFH0M990ca9QySm5QIxUGno1Q__L2Jz1bsUGFaFsON3kTnNjnK7I2yLzj5vb_7U&udm=2&prmd=isvnmbtz&sa=X&sqi=2&ved=2ahUKEwimj4-B4_CFAxVuC0QIHRh4ALQQtKgLegQIExAB&biw=1728&bih=992&dpr=2)

Let's keep the standard name

Reply:
Oh, I see we were using 'trainset' in the old preprocess function when args.data != 'shakespeare'. I am going to change all places to train_dataset